client/webserver/site/src/js/markets.ts

```diff
@@ -2512,13 +2598,10 @@ export default class MarketsPage extends BasePage {
     const avail = note.balance.available
     switch (note.assetID) {
       case mkt.baseCfg.id:
-        // If we're not showing the max order panel yet, don't do anything.
-        if (!mkt.maxSell) break
```

Comment:
Hmm. Why?

Reply:
This way the max order panel and the order button update when the balance does.